src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -15,19 +15,32 @@ interface TracksPropTypes {
     width:number;
     mutationGenePanelIconData?:IKeyedIconData;
     copyNumberGenePanelIconData?:IKeyedIconData;
+    onSelectGenePanel?:(name:string)=>void;
 }
 
-export const DEFAULT_GENOME_BUILD="GRCh37";
-
+interface TracksStateTypes {
+    genePanelInTooltip:string;
+}
 
-export default class Tracks extends React.Component<TracksPropTypes, {}> {
+export const DEFAULT_GENOME_BUILD="GRCh37";
```

Comment:
not sure but i think this constant already exists elsequere.  we should have it one place unless it's only the default for this specific feature

Reply:
I searched through the source code and only found this constant used in this feature, so I think it's alright to keep it here.